Cargo.toml

```diff
@@ -1,36 +1,21 @@
-[package]
-name = "rngs"
-version = "0.0.0"
-authors = ["The Rand Project Developers", "The Rust Project Developers"]
-license = "MIT OR Apache-2.0"
-readme = "README.md"
-repository = "https://github.com/rust-random/rngs"
-description = """
-Extra random number generators
-"""
-autobenches = true
-edition = "2018"
-publish = false
-
 [workspace]
 members = [
+    "benches",
```

Comment:
I think it's better to exclude benches from the root workspace and test its build in a separate CI job.

Reply:
But it *is* part of the workspace.

Reply:
It's our decision whether to include a crate into workspace or not. In `rand` we [exclude](https://github.com/rust-random/rand/blob/master/Cargo.toml#L66) benches because `criterion` is relatively heavy dependency and it's somewhat wasteful to build it during crate tests.

Reply:
Oh, you managed to sneak that change into https://github.com/rust-random/rand/pull/1448

Reply:
And your CI script tests the base repo, not the `benches` package. Check the log:
https://github.com/rust-random/rand/actions/runs/9008465527/job/24750605137

Reply:
No, you can set the default `working-directory`.

...only I thought that didn't work since `cargo fmt --all -- --check` from the `benches` directly still complains about stuff in the root workspace. Cargo bug!

Reply:
Yeah, `defaults.run.working-directory` works as expected. The Clippy complaining was probably because working dir was not changed in the previous version of config. Running it locally in the benches dir does not show warnings for `rand` and `rand_distr`.

Reply:
> The Clippy complaining was probably because working dir was not changed in the previous version of the config.

I fixed that; it alone wasn't enough. And try e.g. `cargo test -p rand_hc` from the `benches` dir; it should not work but it does!

Reply:
Maybe we also need to add `package.workspace = "."` to `benches`' Cargo.toml?

Reply:
Result:
```
$ cargo test
error: root of a workspace inferred but wasn't a root: /home/dhardy/projects/rand/rngs/benches/Cargo.toml
```

Reply:
I think it's just a Cargo bug we have to deal with... lets merge this?

Reply:
Ah,yes. I meant:
```toml
[workspace]
members = ["."]
```